.kokoro/publish_javadoc.sh

```diff
@@ -5,15 +5,13 @@ if [[ -z "${CREDENTIALS}" ]]; then
   CREDENTIALS=${KOKORO_KEYSTORE_DIR}/73713_docuploader_service_account
 fi
 
+pyenv install 3.7.2
```

Comment:
Should we just have it in the image we're using? Do we do it in other repos the same?

Reply:
Having it default in the image is the best way. Other repos aren't setting the pyenv versions. Should I just try running it without setting the version?